src/Template/Element/Form/advanced_properties.twig

```diff
@@ -12,7 +12,7 @@
 
             {% set options = Schema.controlOptions(key, value, schema.properties[key]) %}
 
-            {{ Form.control(key, options)|raw }}
+            {{ Form.control(__(key), options)|raw }}
```

Comment:
Not sure if we want to translate field names... some yes probably, but we have to think about it

Reply:
Fair enough. I'd say that we should at least translate `title` and `description`.